Documented information about sleep and wake namespace (#366)
okteto/docs: src/content/cloud/namespaces.mdx

<p align="center"><Image src="/docs/cloud/images/share.png" alt="share a namespace" width="850" /></p>

In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.

> One of the best way to reduce cost while working is to put your namespace on sleep. A week has 168 hours and let's assume

Reviewer: ```suggestion
One of the best ways to reduce cost while working is to put your namespace in the `sleep` state. Sleeping a namespace scales your development environment down to an idle state to minimize your costs to the absolute minimum, savings dozens of hours per week in cloud costs.

You can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](/self-hosted/administration/configuration/#gc) configuration.
```

@pchico83 I assume this wording is appropriate since the costs won't be "zero". Or would they? I'm unclear on the specifics of whether it's something on our end that we wake the environment when needed and/or if the environment leaves some bare minimum processes in place for the wake operation.

In either case: @NitishKumar06 this is a non-blocking proposed change 😄 I'm proposing this language (and proposing we remove the text below) since docs are typically direct and objective and this copy comes across as more appropriate for a marketing page to promote the value of this feature (sleep/wake)

Reviewer 2: I agree. I would reduce the scope of the PR to document the CLI commands

Author: Thanks! I've removed the text.

> your team works for 40 hours per week. Hence, about 128 hours per week, your team is not working. You can put your workloads

Reviewer: ```suggestion
```

> to sleep for the time team is not working on it and wake the workloads when required.

Reviewer: ```suggestion
```

You can sleep your namespace from the command line with `okteto namespace sleep <name-of-namespace>`.

When you want to wake your namespace, you can use the command `okteto namespace wake <name-of-namespace>`.

You can sleep your namespace from Okteto Cloud itself by clicking on Sleep dialog.

> In order to wake the namespace, you can click on Wake dialog.

Reviewer: ```suggestion
In order to wake the namespace, you can click on Wake dialog.

:::tip

You can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](/self-hosted/administration/configuration/#gc) configuration.

:::

```

Reviewer: @NitishKumar06 Please 😄

If you commit the suggestion I think we're set on this one 👍

Author: completely missed this one! Thanks for reminding me - Changes are applied!

<p align="center"><Image src="/docs/cloud/images/share.png" alt="share a namespace" width="850" /></p>

In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.

> You can sleep your namespace from the command line with `okteto namespace sleep <name-of-namespace>`.

Reviewer 2: ```suggestion
You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
```
That means it's optional

Author: Done!

> When you want to wake your namespace, you can use the command `okteto namespace wake <name-of-namespace>`.

Reviewer 2: ```suggestion
When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
```
That means it's optinal
